performance_benchmarks/servers/cheroot_wsgi.py

```diff
@@ -0,0 +1,17 @@
+"""
+Adapted from https://github.com/cherrypy/cheroot/blob/master/cheroot/test/test_wsgi.py
+"""
+
+from cheroot import wsgi
+
+def simple_wsgi_server():
+    def app(_environ, start_response):
+        status = '200 OK'
+        response_headers = [('Content-type', 'text/plain')]
+        start_response(status, response_headers)
+        return [b'Hello world!']
+    server = wsgi.Server(("localhost", 8080), app, timeout=100)
```

Comment:
Try playing with these args:
```suggestion
    server = wsgi.Server(
        ("localhost", 8080), app,
        numthreads=100, request_queue_size=100,
        accepted_queue_size=100, timeout=100,
    )
```

Reply:
Isn't better to leave most of the args to defaults like in other benchmarked WSGI servers?

Reply:
It depends on what you want to measure, I think. Ideally, all of the servers should use similar sets of settings specified explicitly, otherwise, you're testing them with their defaults which may be based on different assumptions and perform differently just because they target use-cases that have different requirements. It'd probably be reasonable to check all of the benchmarks for this.

Reply:
The benchmarks I've added in this repo were not meant to be very detailed and scientific. They were just a quick comparison between simple "default" single worker server setups (pull an example from the server's documentation and then run a benchmark). I think it's fine to just keep the servers simple.